lib/dynamo.js

```diff
@@ -10,29 +11,32 @@ const clientOptions = { region, httpOptions, maxRetries }
 // NOTE: must use STS in the region containing the lambda, not the DDB table
 // TODO: shouldn't this happen automagically, with clientOptions.region set?
 const myRegion = process.env.AWS_REGION || 'us-east-1'
-const endpoint = new AWS.Endpoint(`sts.${myRegion}.amazonaws.com`)
-const sts = new AWS.STS({ endpoint, httpOptions, maxRetries })
+const sts = new STSClient({ region: myRegion, httpOptions, maxRetries })
```

Comment:
Passing `httpOptions` / `maxRetries` no longer works like this.  You have to instead pass a [requestHandler](https://github.com/PRX/dovetail-cdn-arranger/blob/main/lib/util.js#L40).

For retries, we were just setting to the default, so I dropped the param.  Otherwise you'd have to do the new `maxAttempts` and `retryStrategy` thing.

Reply:
Also note to selves: should confirm that omitting the explicit `endpoint` works in V3 now.  And then remove the `NOTE:` / `TODO:` above concerning that.